tox.ini

```diff
@@ -1,20 +1,20 @@
 [tox]
 envlist =
-    py3{4,5,6}
+    py3{5,6}
     pypy3
 skip_missing_interpreters = True
 
 [testenv]
 deps =
-    cassandra-driver
     flake8
     isort
 commands =
-    flake8 --show-source aiocassandra.py setup.py tests.py
-    isort --check-only aiocassandra.py setup.py tests.py
+    flake8 --show-source aiocassandra.py setup.py
+    isort --check-only aiocassandra.py setup.py
+    flake8 --show-source tests
```

Comment:
Why not append it to the previous command?

Reply:
isort for folders requires `-rc` , as well split lint code and lint tests in different parts